### PR TITLE
Replace current implementation of sending queryParams to client

### PR DESCRIPTION
**Pull request by kenkouot**

### CODE REVIEW

**Comment by mklucsarits** [comment]
Looks ok to me at a quick glance.

File: server/lib/Utils.ts [hunk 1]
```diff
@@ -134,6 +134,21 @@ module Utils {
 		}
 		return null;
 	}
+
+	export function parseQueryParams (obj: any): any {
```
Comment (rogatty) [inline server/lib/Utils.ts]: Looks like a good candidate for unit tests.

File: server/lib/Utils.ts [hunk 2]
```diff
@@ -134,6 +134,21 @@ module Utils {
...
+		var parsed = {},
+			key,
+			prop;
```
Comment (rogatty) [inline server/lib/Utils.ts]: `server/lib/Utils.ts(140,4): 7005 Variable 'key' implicitly has an 'any' type.`
Both of them do.

File: server/lib/Utils.ts (hunk 1 above)
Comment (hakubo) [inline server/lib/Utils.ts]: any (:)) chance for anything better then any?

Comment (hakubo) [inline server/lib/Utils.ts]: and actually doesn't hapi give you query params parsed? - I am sure they do
https://github.com/hapijs/qs

Comment (kenkouot) [inline server/lib/Utils.ts]: @hakubo Parsed to strings, but this casts to int when possible (for obvious, non-silly cases)

**Pull request merged**